include/seqan3/alphabet/structure/dot_bracket3.hpp

```diff
@@ -157,7 +157,7 @@ class dot_bracket3 : public alphabet_base<dot_bracket3, 3>
  * \returns std::vector<seqan3::dot_bracket3>
  *
  * You can use this string literal to easily assign to a vector of seqan3::dot_bracket3 characters:
- * \snippet test/snippet/alphabet/structure/dot_bracket3.cpp string_literal
+ * \include test/snippet/alphabet/structure/dot_bracket3_literal.cpp
```

Comment:
Oh, also here should be the string literal only. Didn't see it in the first review round – sorry!